helm/hpcc/templates/_helpers.tpl

```diff
@@ -2058,3 +2058,31 @@ spec:
 {{- end -}}
 {{- end -}}
 
+{{ define "hpcc.generateWarnings" }}
+{{/*
+Generate a list of warnings
+Pass in dict with root
+*/}}
+{{- $ctx := dict "warnings" list "root" .root -}}
+{{- include "hpcc.getWarnings" $ctx -}}
+{{- if $ctx.warnings -}}
+warnings:
+ {{- range $warning := $ctx.warnings }}
+- msg: {{ $warning.msg | quote }}
+  severity: {{ $warning.severity | quote }}
+  source: {{ $warning.source | quote }}
+ {{- end -}}
+{{- end -}}
+{{- end }}
+
+{{ define "hpcc.printWarnings" }}
```

Comment:
minor: why was the whitespace chomping removed here (in commit 5) ?

All our helper functions have whitespace chomping here to avoid introducing new lines each time they are called